zrml/authorized/Cargo.toml

```
@@ -0,0 +1,36 @@
+[dependencies]
+frame-benchmarking = { branch = "polkadot-v0.9.8", default-features = false, optional = true, git = "https://github.com/paritytech/substrate" }
+frame-support = { branch = "polkadot-v0.9.8", default-features = false, git = "https://github.com/paritytech/substrate" }
+frame-system = { branch = "polkadot-v0.9.8", default-features = false, git = "https://github.com/paritytech/substrate" }
+parity-scale-codec = { default-features = false, features = ["derive"], version = "2.0" }
+sp-runtime = { branch = "polkadot-v0.9.8", default-features = false, git = "https://github.com/paritytech/substrate" }
+zeitgeist-primitives = { default-features = false, path = "../../primitives" }
+zrml-market-commons = { default-features = false, path = "../market-commons" }
+
+[dev-dependencies]
+pallet-balances = { branch = "polkadot-v0.9.8", git = "https://github.com/paritytech/substrate" }
+pallet-timestamp = { branch = "polkadot-v0.9.8", git = "https://github.com/paritytech/substrate"}
+sp-io = { branch = "polkadot-v0.9.8", git = "https://github.com/paritytech/substrate" }
+
+[features]
+default = ["std"]
+runtime-benchmarks = [
+    "frame-benchmarking",
+    "frame-support/runtime-benchmarks",
+    "frame-system/runtime-benchmarks"
+]
+std = [
+    'frame-benchmarking?/std',
+    'frame-support/std',
+    'frame-system/std',
+    'parity-scale-codec/std',
+    'sp-runtime/std',
+    'zeitgeist-primitives/std',
+    'zrml-market-commons/std'
+]
+
+[package]
+authors = ["Zeitgeist PM <contact@zeitgeist.pm>"]
+edition = "2018"
+name = "zrml-authorized"
+version = "0.1.0"
```

Let's start where all the other pallets are

```suggestion
version = "0.1.2"
```